HowToFIDO.md

```diff
@@ -670,10 +686,13 @@ navigator.credentials.create({
     rp: {...},
     user: {...},
     challenge: ...,
-    pubKeyCredParams: {
+    pubKeyCredParams: [{
       type: "public-key",
       alg: -7
-    },
+    }, {
+      type: "public-key",
+      alg: -257
```

Comment:
There are other places besides here (and the initial UVPA scenario) where pubKeyCredParams is incorrectly represented as an object and you've corrected to an array (with additional alg). May I suggest that if you're going to fix it in two places, how about fixing it in all 5 places :)